lib/katello/tasks/reimport.rake

```diff
@@ -35,7 +41,10 @@ namespace :katello do
 
     models << Katello::OstreeBranch if Katello::RepositoryTypeManager.find(Katello::Repository::OSTREE_TYPE).present?
     models << Katello::Deb if Katello::RepositoryTypeManager.find(Katello::Repository::DEB_TYPE).present?
-
+    ignore_content_types.each do |model|
+      puts "\e[33mIgnoring pulp3 enabled content type : #{model.name}\e[0m\n"
+    end
+    models -= ignore_content_types
```

Comment:
Can we use the RepositoryTypeManager to get rid of the list of content models?  I think there are some that are missing.

Something like (haven't checked actual syntax):
```
models = [ 
              Katello::Subscription,
              Katello::Pool,
              Katello::Content
]
Katello::RepositoryTypeManager.repository_types.each |repo_type|
    if SmartProxy.pulp_master.pulp3_support(repo_type)
       puts "Ignoring types: #{repo_type.indexable_content_types}"
   else
      models += repo_type.indexable_content_types
   end
end
```

Reply:
that could also get rid of the check for ostree and deb_type, since it would be built into the 'loop'

Reply:
Updated with the changes as you pointed. Added Katello::ContentViewPuppetEnvironment to list of models to import.